src/commcare_cloud/commands/ansible/ansible_playbook.py

```diff
@@ -190,18 +192,19 @@ def run(self, args, unknown_args):
             args, unknown_args, always_skip_check=always_skip_check)
 
 
-class UpdateConfig(_AnsiblePlaybookAlias):
+class UpdateConfig(CommandBase):
     command = 'update-config'
     help = """
     Run the ansible playbook for updating app config.
 
     This includes django `localsettings.py` and formplayer `application.properties`.
     """
 
+    arguments = (shared_args.BRANCH_ARG,)
+
     def run(self, args, unknown_args):
-        args.playbook = 'deploy_localsettings.yml'
-        unknown_args += ('--tags=localsettings',)
-        return AnsiblePlaybook(self.parser).run(args, unknown_args)
+        return commcare_cloud(args.env_name, 'ansible-playbook', 'deploy_localsettings.yml',
+                              tags='localsettings', branch=args.branch, *unknown_args)
```

Comment:
Why did you decide to handle the `--branch` argument this way, but didn't add any of the other `ansible-playbook` args?

https://github.com/dimagi/commcare-cloud/blob/9639f0cde5e081318c1405d3270fac8819fc30cb/src/commcare_cloud/commands/ansible/ansible_playbook.py#L38-L43

When I run `cchq production update-config --help` I only see documentation for two arguments:
```
optional arguments:
  -h, --help
  --branch BRANCH
```
even though others are applicable and will be passed through to the underlying `ansible-playbook` command if provided.

I suppose one (not very elegant) way to work around this would be to include a note in the help string that all optional arguments for the `ansible-playbook` command are also accepted.